.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -30,9 +30,9 @@ Discord username (if different from GitHub):
 <!-- DO NOT DELETE THIS -->
 ## How to download the PR for QA
 
-Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.
+Every new commit to this PR will cause a new A32NX & A380X artifact to be created, built, and uploaded.
```

Comment:
Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.